recipe/meta.yaml

```diff
@@ -1,21 +1,25 @@
-{% set version = "1.1.1" %}
-{% set sha256 = "14e7ff1b87c2cb901f3a6bebb64cbc5e8d624a067319c242b55cf77c341018fc" %}
+{% set version = "1.2.0rc1" %}
+{% set sha256 = "fd403983766e9e9e723de4a7f07d05d4bda59a822f171a6b6fc3012e51a13a8c" %}
 
 package:
   name: obspy
   version: {{ version }}
 
 source:
   fn: obspy-{{ version }}.zip
-  url: https://pypi.io/packages/source/o/obspy/obspy-{{ version }}.zip
-  # url: https://github.com/obspy/obspy/files/2872202/obspy-1.1.1rc4.zip
+  # url: https://pypi.io/packages/source/o/obspy/obspy-{{ version }}.zip
+  url: https://github.com/obspy/obspy/files/4108546/obspy-1.2.0rc1.zip
   sha256: {{ sha256 }}
 
+# for RC builds
+channel_targets:
+  - conda-forge rc_obspy
```

Comment:
This has to go into `conda_build_config.yaml`.
Also note that `obspy_rc` would be the preferred naming, we have a pending change for that: https://github.com/conda-forge/cfep/pull/3#issuecomment-578364893

Reply:
(see also https://github.com/conda-forge/cfep/pull/21 reflecting the preferred naming)

Reply:
Thanks! However, we don't actually intend to merge (and thus deploy) the RCs, I was just making sure the builds look good in CI and we do not have packaging problems. We're expecting to push out the release in the next few days and I'll update this PR with the release zipball.